awx/main/tasks/system.py

```diff
@@ -684,8 +684,13 @@ def awx_receptor_workunit_reaper():
     jobs_with_unreleased_receptor_units = UnifiedJob.objects.filter(work_unit_id__in=unit_ids).exclude(status__in=ACTIVE_STATES)
     for job in jobs_with_unreleased_receptor_units:
         logger.debug(f"{job.log_format} is not active, reaping receptor work unit {job.work_unit_id}")
-        receptor_ctl.simple_command(f"work cancel {job.work_unit_id}")
-        receptor_ctl.simple_command(f"work release {job.work_unit_id}")
+        try:
+            receptor_ctl.simple_command(f"work cancel {job.work_unit_id}")
+            receptor_ctl.simple_command(f"work release {job.work_unit_id}")
+        except Exception as e:
+            # leave the cleaning of work unit to administrative_workunit_reaper
```

Comment:
`administrative_workunit_reaper` and `awx_receptor_workunit_reaper` handle different types of work units.

if releasing fails here, i don't believe administrative_workunit_reaper will release them

Reply:
Yes probably is better force release here the workunit.

In administrative_workunit_reaper i need to remove Force release?